src/oauth-handler.js

```diff
@@ -31,7 +32,9 @@ export default function oauth(Client, {
   shipCache = null
 }) {
   function getURL(req, url, qs = { token: req.hull.token }) {
-    return `https://${req.hostname}${req.baseUrl}${url}?${querystring.stringify(qs)}`;
+    const host = `https://${req.hostname}${req.baseUrl}${url}`;
+    if (qs === false) return host;
```

Comment:
Hm, but when you add the `&reset=true`, that would be the dashboard panes, so the url without token won't work (will be unauthorized). But I think it's not something we should have very long discussion, I will let you decide.